e2e-disabled/TransactionHttp.spec.ts

```diff
@@ -122,10 +119,10 @@ describe("TransactionHttp", () => {
     const transactionHttp = new TransactionHttp([{domain: TestVariables.DEFAULT_TEST_DOMAIN}]);
     const account = Account.createWithPrivateKey(privateKey);
 
-    const transferTransaction = TransferTransaction.createWithMosaics(
-      TimeWindow.createWithDeadline(),
+    const transferTransaction = TransferTransaction.create(
       new Address(recipientAccount),
-      [new XEM(10)],
+      TxType.other,
+      new XEM(10),
       EmptyMessage);
     const signedTransaction = account.signTransaction(transferTransaction);
     // transactionHttp.announceTransaction(signedTransaction).subscribe(announceSuccessResult => {
```

Comment:
i'm hesistant to publish any commented code in the public repo

Reply:
This is commented from their tests, haven't touched it yet